.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
https://user-images.githubusercontent.com/38065883/174634551-b444df9a-3ebf-4a33-8309-88b3f600f8ae.mov

Use effect that isn't change layout, like the one below

https://codepen.io/lucy_wheel/pen/VxYzKP

Reply:
https://user-images.githubusercontent.com/38065883/174634775-fa71ca4f-babf-472e-bfbc-d87e355492ed.mov

Form shouldn't be sent if all of its field are empty

Reply:
https://user-images.githubusercontent.com/38065883/174635016-ee499fec-c0c7-4cc6-8b0d-639a449cfde2.mov

Page isn't scrollable from the very beginning, cause its already has `with-menu` class for some reason